inc/namespace.php

```diff
@@ -108,7 +108,7 @@ function register_column( array $columns ) : array {
 	// to insert Tasks at the correct position.
 	foreach ( $columns as $key => $value ) {
 		$new_columns[ $key ] = $value;
-		if ( $key === 'tags' ) {
+		if ( $key === apply_filters('altis.publication-checklist.show_tasks_column_after', 'tags' ) {
```

Comment:
```suggestion
		if ( $key === apply_filters( 'altis.publication-checklist.show_tasks_column_after', 'tags' ) {
```

Adding some Nacin spacing 😁 

I think this filter is good to have but can you move it outside of the loop? No point running it for each column. Can you also add a docblock for the filter when you move it up too?

It's best to avoid having filters directly in `if` statements etc too, it's not too readable and you can't add docblocks nicely.

Reply:
Also not really required but it might be convenient to pass the current post type in this filter as a 2nd parameter given the choice of column may differ.